tests/test_midpointsedges.py

```diff
@@ -0,0 +1,43 @@
+import numpy as np
+
+import meshzoo
+
+
+def test_midpoints_edges_tri():
+    points, cells = meshzoo.rectangle_tri(a0=(0, 0), a1=(1, 1), n=2)
+    points_new, cells_new = meshzoo.insert_midpoints_edges(
+        points, cells, cell_type="triangle"
+    )
+    assert len(points_new) == 9
```

Comment:
Let's assert the same sizes _before_ `insert_midpoints_edges` too. This way, one can look at the test an see what has changed exactly.